clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/security/KubernetesNamedAccountCredentials.groovy

```diff
@@ -213,7 +213,9 @@ public class KubernetesNamedAccountCredentials implements AccountCredentials<Kub
       }
 
       return new KubernetesCredentials(
-          new KubernetesApiAdaptor(name, config, spectatorRegistry),
+          new KubernetesApiAdaptor(name, config,
+            new KubernetesApiClientConfig(kubeconfigFile, context, cluster, user, namespaces, serviceAccount),
```

Comment:
I'd pull this constructor invocation onto a separate line, this is becoming a little dense/hard to parse